### PR TITLE
adherence calcs; skip unnecessary update

### PR DESCRIPTION
**Pull request by sravfeyn**
@mkangia

### CODE REVIEW

File: custom/enikshay/tasks.py [hunk 1]
```diff
@@ -63,32 +73,36 @@ def run(self):
...
-                logger.error(
-                    "Error calculating updates for episode case_id({}): {}".format(
-                        episode.case_id,
-                        e
-                    )
-                )
         logger.info(
-            "Summary of enikshay_task: Updated {updates}, errored on {errors} and {noupdates} "
-            "cases didn't need update ".format(updates=update_count, errors=error_count, noupdates=noupdate_count)
+            "Summary of enikshay_task: domain: {domain}, duration (sec): {duration} "
```
Comment (mkangia) [inline custom/enikshay/tasks.py]: Whats the need for using `domain` here? Can this be used for a domain except `enikshay`?

File: custom/enikshay/tasks.py [hunk 2]
```diff
@@ -45,6 +45,16 @@ def enikshay_task():
         updater.run()
 
 
+class Timer:
```
Comment (mkangia) [inline custom/enikshay/tasks.py]: Nice!

**Review by mkangia — APPROVED** [review]
Overall looks good. Just in context of domain, if this can't be used for any domain other than 'enikshay' we might want to restrict it to that. Can skip it if that is unnecessary.

File: custom/enikshay/tasks.py (hunk 1 above)
Comment (sravfeyn) [inline custom/enikshay/tasks.py]: This is run for only adherence_task [toggle enabled domains](https://github.com/dimagi/commcare-hq/blob/73c71b6cc753c0ea949596f290fbd6d650a158ab/custom/enikshay/tasks.py#L42), there are couple of enikshay test domains that will also need to have this run

**Pull request merged**